example.main.scss

```diff
@@ -23,6 +23,8 @@
  * Global...............Project-wide variables and settings.
  *
  * TOOLS
+ * Rem..................A function to convert a pixel input into a rem
```

Comment:
There's a mixin too. So something along the lines "A function and a mixin [...]" is more precise.